citi-onboarding/Recorda-one-page
# feat(news_CRUD): created the 'News' database

### **What I did:**
- Created the News model
- Created the NewsController
- Created the CRUD routes to the NewsController
- Cleaned some visual mess

### **How to test**
- git checkout main
- git pull
- git checkout feature/news_CRUD
- git pull
- be sure you are on the server directory
- write "docker compose up" at the terminal
- test the database with "get", "post", "delete" and "put" at Insomnia
- check if everything is working on Postbird
